test/controllers/OpenPaymentsControllerSpec.scala

```diff
@@ -30,6 +30,7 @@ import org.jsoup.Jsoup
 import org.jsoup.nodes.Document
 import play.api.http.Status
 import play.api.mvc.{Request, Result}
+import play.api.test.FakeRequest
```

Comment:
I think this import is unused as all tests in this Spec that are using a FakeRequest are using ones defined in ControllerBaseSpec